src/wallet/rpcwallet.cpp

```diff
@@ -537,6 +538,89 @@ static RPCHelpMan sendtoaddress()
     };
 }
 
+static RPCHelpMan walletestimatefee()
+{
+    return RPCHelpMan{"walletestimatefee",
+        "\nEstimates the fee per kilobyte needed for a transaction to begin from this wallet\n"
```

Comment:
"Estimates the fee per kilobyte needed for a transaction to be generated from this wallet, taking into account mempool minfee and fallback fee. In contrast, `estimatesmartfee` estimates ignores these are returns a pure fee estimate based on prior block confirmations."

something like this? For parameters let the param docs explain what they do.

You can keep the description short and let the documentation of the arguments do the explanation of the rest.